docs/bir-spec/src/test/resources/testng.xml

```diff
@@ -24,7 +24,7 @@ under the License.
     <!-- Ballerina BIR Spec Test Cases. -->
     <test name="ballerina-bir-spec-test-suite" parallel="false">
         <classes>
-            <class name="org.ballerinalang.birspec.BIRSpecTest"/>
+<!--            <class name="org.ballerinalang.birspec.BIRSpecTest"/>-->
```

Comment:
Disabling temporarily, since it fails ATM. @pcnfernando is looking into it.

Reply:
Enabled through 8a86a532a7327f8f2e3cc5a66f0f68c9433aaf19